keps/sig-auth/3221-structured-authorization-configuration/README.md

```diff
@@ -0,0 +1,802 @@
+
+None other than what we are planning to add as part of the feature.
+
+### Dependencies
+
+<!--
+This section must be completed when targeting beta to a release.
+-->
+
+###### Does this feature depend on any specific services running in the cluster?
+
+<!--
+Think about both cluster-level services (e.g. metrics-server) as well
+as node-level agents (e.g. specific version of CRI). Focus on external or
+optional services that are needed. For example, if this feature depends on
+a cloud provider API, or upon an external software-defined storage or network
+control plane.
+
+For each of these, fill in the following—thinking about running existing user workloads
+and creating new ones, as well as about cluster-level services (e.g. DNS):
+  - [Dependency name]
+    - Usage description:
+      - Impact of its outage on the feature:
+      - Impact of its degraded performance or high-error rates on the feature:
+-->
+
+None
+
+### Scalability
+
+> Note: This section is good-to-have for Alpha.
+
+###### Will enabling / using this feature result in any new API calls?
+
+No. No additional calls will be made to the Kubernetes API Server.
+
+###### Will enabling / using this feature result in introducing new API types?
+
+Yes, but these are types for defining configuration and are not served.
+
+###### Will enabling / using this feature result in any new calls to the cloud provider?
+
+No.
+
+###### Will enabling / using this feature result in increasing size or count of the existing API objects?
+
+No.
+
+###### Will enabling / using this feature result in increasing time taken by any operations covered by existing SLIs/SLOs?
+
+Maybe, the only case where API Server requests would be impacted is when the
+cluster administrator defines multiple webhooks.
+
+**Note**: This is a result of the intended feature.
+If multiple webhooks are defined and one or more of them are unreachable, the
+request latency would get a hit but this is upto the configuration made by the
+user. The feature implementation itself doesn't introduce any change to the
+existing SLIs/SLOs.
+
+###### Will enabling / using this feature result in non-negligible increase of resource usage (CPU, RAM, disk, IO, ...) in any components?
+
+<!--
+Things to keep in mind include: additional in-memory state, additional
+non-trivial computations, excessive access to disks (including increased log
+volume), significant amount of data sent and/or received over network, etc.
+This through this both in small and large cases, again with respect to the
+[supported limits].
+
+[supported limits]: https://git.k8s.io/community//sig-scalability/configs-and-limits/thresholds.md
+-->
+
+No. There would be very minimal addition to the memory used by the API Server and
+number of log entries written to the disk. During the Alpha implementation,
+the small impact will be measured and rationalized to keep the addition
+minimal. The addition would be well within the scalability limits and
+thresholds.
```

Comment:
I would argue that this feature could make common use cases of a single webhook even faster by introducing the CEL expression filtering to limit the number of webhook calls.

Reply:
Added a note.